org.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.seventhstring.Transcribe
```

Comment:
seventhstring.org redirect to seventhstring.com, and I think it's more appropriate to use that unless upstream already use something else.

Reply:
will do!

Reply:
done